src/Relude/List.hs

```diff
@@ -63,14 +65,33 @@ infix 9 !!?
     go _ []     = Nothing
 {-# INLINE (!!?) #-}
 
+{- | '!!?' with its arguments flipped.
+
+Get element from list using index value starting from `0`.
+
+>>> maybeAt 0 []
+Nothing
+
+>>> maybeAt 3 ["a", "b", "c"]
+Nothing
+
+>>> maybeAt (-1) [1, 2, 3]
+Nothing
+
+>>> maybeAt 2 ["a", "b", "c"]
+Just "c"
+
+-}
+maybeAt :: Int -> [a] -> Maybe a
+maybeAt = flip (!!?)
+{-# INLINE maybeAt #-}
+
 {- | Partitions a list based on the result of function which produces an Either value. List of all elements producing Left are extracted, in order, to the first element of the output tuple. Similarly, a list of all elements producing Right are extracted to the second element of output.
 
->>> :{
- divideEvenOrShow :: Int -> Either Int String
- divideEvenOrShow n
-     | even n = Left $ n `div` 2
-     | otherwise = Right $ "Odd: " <> show n
- :}
+> divideEvenOrShow :: Int -> Either Int String
+> divideEvenOrShow n
+>     | even n = Left $ n `div` 2
+>     | otherwise = Right $ "Odd: " <> show n
```

Comment:
I think formatting got wrong here 🤔 
I believe, this is the reason why tests are failing on CI

Reply:
Ah, actually I changed that because curly braces were being displayed on haddocks and thought it was weird; didn't realize that it was used in tests. I reverted it to the way it was, thanks!